### PR TITLE
Navigation through siblings

### PR DESCRIPTION
**Pull request by hermescnp**
Now in the app is possible to travel between sibling instances (Not only travel to a child).Then some new instances were added to zonification and modified in the 3d model.

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -74,7 +74,7 @@ h1 {
 h2 {
   font-weight: 300;
   font-size: 20px;
-  margin: 0px 0px 5px 0px;
+  margin: 0px 0px 0px 0px;
```
Comment (nminaya) [inline app/globals.css:77]: What about `margin: 0;`?

File: components/Experience/PathAnalyzer.tsx [hunk 2]
```diff
@@ -24,33 +24,34 @@ export function isPathEquivalent(pathA : THREE.CatmullRomCurve3, pathB : THREE.C
...
-      }
-  
-      parentKey = findParentKey(data, parentKey);
+  const numSamples = 100
+
+  let closestDistanceSquared = Infinity;
+  let closestT = 0;
+
+  for (let i = 0; i <= numSamples; i++) {
+    const t = i / numSamples;
```
Comment (nminaya) [inline components/Experience/PathAnalyzer.tsx:38]: Can we rename `t` to a more descriptive name?

**Review by nminaya — COMMENTED** [review]
@hermescnp I've left some minor comments

File: components/Experience/PathAnalyzer.tsx (hunk 2 above)
Comment (hermescnp) [reply components/Experience/PathAnalyzer.tsx:38]: the “t” value is a global used variable to refer to a number between 0 and 1 that is needed in a LERP math function to return the interpolated point as result, so in this compontent im trying to get the closest “t” value to aproximate the point on curve to the given instance position. So in this case, the name maker clearer what does the function do.

File: app/globals.css (hunk 1 above)
Comment (hermescnp) [reply app/globals.css:77]: true, i didnt notice. Thanks!

**Pull request merged**